Source/bmalloc/bmalloc/IsoMallocFallback.cpp

```diff
@@ -61,7 +61,7 @@ void determineMallocFallbackState()
 
 MallocResult tryMalloc(
     size_t size,
-    CompactAllocationMode mode
+    [[maybe_unused]] CompactAllocationMode mode
```

Comment:
We use `UNUSED_PARAM(mode)` in the body.

Reply:
There was some discussion on the mailing list recently and my understanding was we want to move toward `[[maybe_unused]]` instead of the `UNUSED_PARAM` macro. Granted, there was no definitive stance taken so it could go either way!

Reply:
https://lists.webkit.org/pipermail/webkit-dev/2024-January/032609.html for reference.